ktor-server/ktor-server-plugins/ktor-server-autoreload/api/ktor-server-autoreload.api

```diff
@@ -0,0 +1,23 @@
+public final class io/ktor/server/autoreload/ApplicationEngineEnvironmentReloading : io/ktor/server/engine/ApplicationEngineEnvironment {
+	public static final field Companion Lio/ktor/server/autoreload/ApplicationEngineEnvironmentReloading$Companion;
+	public fun <init> (Ljava/lang/ClassLoader;Lorg/slf4j/Logger;Lio/ktor/server/config/ApplicationConfig;Ljava/util/List;Ljava/util/List;Ljava/util/List;Lkotlin/coroutines/CoroutineContext;Ljava/lang/String;)V
+	public synthetic fun <init> (Ljava/lang/ClassLoader;Lorg/slf4j/Logger;Lio/ktor/server/config/ApplicationConfig;Ljava/util/List;Ljava/util/List;Ljava/util/List;Lkotlin/coroutines/CoroutineContext;Ljava/lang/String;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun <init> (Ljava/lang/ClassLoader;Lorg/slf4j/Logger;Lio/ktor/server/config/ApplicationConfig;Ljava/util/List;Ljava/util/List;Ljava/util/List;Lkotlin/coroutines/CoroutineContext;Ljava/lang/String;Z)V
+	public synthetic fun <init> (Ljava/lang/ClassLoader;Lorg/slf4j/Logger;Lio/ktor/server/config/ApplicationConfig;Ljava/util/List;Ljava/util/List;Ljava/util/List;Lkotlin/coroutines/CoroutineContext;Ljava/lang/String;ZILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun getApplication ()Lio/ktor/server/application/Application;
+	public fun getClassLoader ()Ljava/lang/ClassLoader;
+	public fun getConfig ()Lio/ktor/server/config/ApplicationConfig;
+	public fun getConnectors ()Ljava/util/List;
+	public fun getDevelopmentMode ()Z
+	public fun getLog ()Lorg/slf4j/Logger;
+	public fun getMonitor ()Lio/ktor/events/Events;
+	public fun getParentCoroutineContext ()Lkotlin/coroutines/CoroutineContext;
+	public fun getRootPath ()Ljava/lang/String;
+	public final fun reload ()V
+	public fun start ()V
+	public fun stop ()V
+}
+
+public final class io/ktor/server/autoreload/ApplicationEngineEnvironmentReloading$Companion {
```

Comment:
while we anyway brake binary compatibility here, may be just remove empty companion?

Reply:
I'm going to fix binary compatibility